meterfs/test_meterfs_writeread.c

```diff
@@ -27,6 +29,24 @@ static void cleanBuffs(void)
 }
 
 
+static void turnCheck(int fd, file_info_t *info, char *bufftx, char *buffrx, unsigned int iterNum)
+{
+	int i;
+
+	if (iterNum >= 10000 || info->recordsz < 4)
+		return;
+
+	for (i = 0; i < iterNum; ++i) {
+		snprintf(bufftx, sizeof(bufftx), "%04d", i); 
```

Comment:
For 32bit processor `sizeof(bufftx) == 4` and the format `"04d"` indicates 5 characters ( 4 gigits + '\0`).